### ISSUE TITLE
Date always reverts to current date

### ISSUE THREAD

**Issue by GoogleCodeExporter** [evancharlton/android-mileage#11]
```
What steps will reproduce the problem?
1. Enter new fillup
2. Change date to past date
3. Hit save

Expected behavior:
Fillup saved with entered date

Actual behavior:
Fillup saved with today's date.

Mileage version: 3.0.1
Android version:
Device:

Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `evancharlton` on 4 Mar 2011 at 3:59

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.0.2
```

Original comment by `evancharlton` on 4 Mar 2011 at 4:06
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `evancharlton` on 17 Apr 2011 at 8:21
- Changed state: **Released**
